java/runtime/src/main/java/org/ray/runtime/util/StringUtils.java

```diff
@@ -0,0 +1,9 @@
+package org.ray.runtime.util;
+
+public class StringUtils {
+
+  public static boolean isNullOrEmpty(String str) {
```

Comment:
I believe this util method is already in the apache common library.

Reply:
Can use `StringUtil` from apache commons, or `Strings` from guava